features/bevragen/persoon/rni/dev/overzicht-gba.feature

```diff
@@ -153,6 +157,38 @@ Rule: RNI-deelnemer gegevens die horen bij categorie 01 (Persoon), 04 (Nationali
     | locatie             | locatiebeschrijving (12.10) | woonboot | locatiebeschrijving | verblijfplaats.verblijfadres.land                |
     | adres               | straatnaam (11.10)          | Spui     | straat              | verblijfplaats.verblijfadres.locatiebeschrijving |
 
+  Abstract Scenario: persoon heeft RNI-deelnemer gegevens voor categorie 08 en <fields> wordt gevraagd
+    Gegeven de persoon met burgerservicenummer '000000036' heeft de volgende 'verblijfplaats' gegevens
+    | naam                         | waarde                               |
+    | straatnaam (11.10)           | Spui                                 |
```

Comment:
straat moet in een 'adres' stap

Reply:
straat verwijderd, is niet noodzakelijk voor dit scenario